lib/system/deploy.rb

```diff
@@ -13,12 +15,36 @@ class Info
       attr_reader :revision, :deployed_at
       attr_reader :release
 
+      delegate :minor_version, :major_version, to: :version
+
       def initialize(info)
         @revision = info.fetch('revision') { `git rev-parse HEAD 2> /dev/null`.strip }
-        @release = ENV['AMP_RELEASE'].presence
+        @release = info.fetch('release') { '2.x' }
```

Comment:
So 2.0.0 is because of SaaS. It does not make sense to have SaaS related configuration here, so let just put it correctly in the deploy repo